src/core/webidl.js

```diff
@@ -809,9 +683,9 @@ function findDfn(parent, name, definitionMap, type, idlElem) {
     }
   }
   if (dfns.length > 1) {
-    const msg = `Multiple \`<dfn>\`s for \`${originalName}\` ${originalParent
-      ? `in \`${originalParent}\``
-      : ""}`;
+    const msg = `Multiple \`<dfn>\`s for \`${originalName}\` ${
+      originalParent ? `in \`${originalParent}\`` : ""
+    }`;
     pub("error", new Error(msg));
   }
   if (dfns.length === 0) {
```

Comment:
Nit: _micro-optimization_: `else if`

Reply:
Going to leave the bits that are already there... as there is a lot of bad code in there that needs to change.